core/table.ts

```diff
@@ -121,6 +126,11 @@ export class Table {
     return this;
   }
 
+  public sqldatawarehouse(sqldatawarehouse: dataform.ISQLDataWarehouseOptions) {
```

Comment:
suggest camel-casing like `sqlDataWarehouse`

Reply:
i think i fixed this everywhere in the code, except in the config. The config would look like this:
```
config {
    type: "table",
    sqldatawarehouse: {
        distribution: "round_robin"
    },
    dependencies:["example"]
}
SELECT 1 AS test
```
I think I did it to be consistent with "bigquery": https://docs.dataform.co/guides/warehouses/bigquery/

But, let me know if you want me to change it.